Source/Scene/Camera.js

```diff
@@ -2166,6 +2166,69 @@ define([
     };
 
     /**
+     * Sets the camera so that the current view completely contains the provided bounding sphere.
+     *
+     * @param {BoundingSphere} boundingSphere The bounding sphere to view.
+     * @param {Object} [options] Object with the following properties:
```

Comment:
None of the camera stuff is finished, I need @bagnell's help to flesh everything out.  Ignore Camera.js for now.